src/material/core/tokens/_m3-system.scss

```diff
@@ -11,16 +14,74 @@ $_system-fallback-prefix: mat-app;
 // Default system level prefix to use when directly calling the `system-level-*` mixins
 $_system-level-prefix: sys;
 
-// Emits CSS variables for Material's system level values. Uses the
-// namespace prefix in $_system-fallback-prefix.
-// e.g. --mat-app-surface: #E5E5E5
-@mixin theme($theme, $overrides: ()) {
-  @include system-level-colors($theme, $overrides, $_system-fallback-prefix);
-  @include system-level-typography($theme, $overrides, $_system-fallback-prefix);
-  @include system-level-elevation($theme, $overrides, $_system-fallback-prefix);
-  @include system-level-shape($theme, $overrides, $_system-fallback-prefix);
-  @include system-level-motion($theme, $overrides, $_system-fallback-prefix);
-  @include system-level-state($theme, $overrides, $_system-fallback-prefix);
+/// Emits necessary CSS variables for Material's system level values for the values defined in the
+/// config map. The config map can have values color, typography, and/or density.
+///
+/// If the config map's color value is an Angular Material color palette, it will be used as the
+/// primary and tertiary colors with a light theme type. Otherwise if the color value is a map,
+/// it must have a `primary` value containing an Angular Material color palette, and optionally
+/// a different `tertiary` palette (defaults to primary palette) and `theme-type` that is either
+/// `light` or `dark` (defaults to light). Color variable definitions will not be emitted if there
+/// are no color values in the config.
+///
+/// If the config map's typography value is a font family string, it will be used as the
+/// plain and brand font family with default bold, medium, and regular weights of 700, 500, and 400,
+/// respectfully. Otherwise if the typography value is a map, it must have a `plain-family` font
+/// family value, and optionally a different `brand-family` font family (defaults to the plain
+/// value) and weights for `bold-weight` (default: 700), `bold-weight` (default: 500), and
+/// `bold-weight` (default: 400). Typography variable definitions will not be emitted if there are
+/// no typography values in the config.
+///
+/// If the config map's density value is a number, it will be used as the density scale. Otherwise
+/// if the density value is a map, it must have a `scale` value. Density variable definitions will
+/// not be emitted if there are no density values in the config.
+///
+/// The application variables emitted use the namespace prefix "--mat-app".
+/// e.g. --mat-app-surface: #E5E5E5
```

Comment:
Wouldn't `$_system-level-prefix` need to also be `mat-app` instead of `sys`?

Reply:
It doesnt align with our standard `mat` prefix that we have for other things, but instead can easily interfere with apps that already use `sys` as a prefix